web/ui/react-app/src/utils/index.ts

```diff
@@ -269,3 +269,13 @@ export const parsePrometheusFloat = (value: string): string | number => {
     return Number(value);
   }
 };
+
+export function debounce<Params extends any[]>(func: (...args: Params) => any, timeout: number): (...args: Params) => void {
```

Comment:
This is producing some compilation errors on CircleCI due to the restrictive TypeScript settings:

```
Failed to compile.

src/utils/index.ts
  Line 273:41:  Unexpected any. Specify a different type  @typescript-eslint/no-explicit-any
  Line 273:75:  Unexpected any. Specify a different type  @typescript-eslint/no-explicit-any
```

Reply:
(you should also be able to run `make ui-lint` locally)

Reply:
Done. When I ran `make ui-lint`, it showed `@typescript-eslint/no-explicity-any` warnings at many places. I guess they are written before the CI job was introduced.